src/openrct2/object/ObjectRepository.cpp

```diff
@@ -709,7 +709,7 @@ extern "C"
                     Object * object = objectRepository->LoadObject(ori);
                     if (object != nullptr)
                     {
-                        StexObject * stexObject = static_cast<StexObject*>(object);
+                        StexObject * stexObject = dynamic_cast<StexObject*>(object);
```

Comment:
Why change it to `dynamic_cast` if you aren't going to null check?

Reply:
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!